v21.2/deploy-app-netlify.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Deploy a Netlify App Built on CockroachDB
+summary: Learn how to use Netlify and CockroachDB Serverless.
+toc: true
+twitter: false
+referral_id: docs_netlify
+---
```

Comment:
Don't forget to add `docs_area`: https://github.com/cockroachdb/docs/blob/master/CONTRIBUTING.md#pages

Reply:
Done!